### PR TITLE
Extra long time for slower providers in SOAP prov testing

### PR DESCRIPTION
**Pull request by mfalesni**
{{pytest: cfme/tests/test_soap.py -v -k provision --long-running}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 378c9cc187e35fa35d2f8e7f7aba7b3cd06a8c3c:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/test_soap.py [hunk 1]
```diff
@@ -460,6 +460,10 @@ def test_vm_tagging(self, soap_client):
 
 
 class TestProvisioning(object):
+    WAIT_TIME = 300                         # For the VM status checking functions
+    WAIT_TIME_SLOW = 600                    # For slow providers
+    SLOW_PROVIDERS = {"rhevm", "scvmm"}     # That is what I was speaking about
```
Comment (seandst) [inline cfme/tests/test_soap.py]: Kill the inline comments, please. Also, I have no idea what you were speaking about.

**Comment by dajoRH** [comment]
Lint report for commit 54121f2fba7f6ea9d5dd12338c2df58877f6d091:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: cfme/tests/test_soap.py [hunk 2]
```diff
@@ -460,6 +460,10 @@ def test_vm_tagging(self, soap_client):
 
 
 class TestProvisioning(object):
+    WAIT_TIME = 300
+    WAIT_TIME_SLOW = 600
+    SLOW_PROVIDERS = {"rhevm", "scvmm"}
+
```
Comment (seandst) [inline cfme/tests/test_soap.py]: Crap, I was unclear. Sorry about that. I liked the comments, just not that they were inline.

Comment (mfalesni) [inline cfme/tests/test_soap.py]: Too late. I hope the constants' names are self-explanatory though.

Comment (seandst) [inline cfme/tests/test_soap.py]: Yeah, they definitely are. Good point.

**Pull request merged**